### PR TITLE
Leaderboard page ranks books and lists number of members who have rea…

### LINKED ISSUE #146: Update Leaderboard Book List to include how many devBetter folks have read it
Should look basically like the leaderboard. Make it a table. Add a column "Ready By dB Member Count" and order it by the count, desc, by default.

### PR DESCRIPTION
**Pull request by ilyanaDev**
…d them. It also includes a link to purchase the book.

Fixes #146

### CODE REVIEW

File: src/DevBetterWeb.Web/Pages/Leaderboard/Index.cshtml [hunk 1]
```diff
@@ -35,20 +36,42 @@
...
+                        </td>
+                        <td>
+                            <a href="@book.PurchaseUrl" target="_blank"><i class="fas fa-shopping-cart"></i></a>
+                        </td>
+                        <td></td>
+                        <td>
+                            @book.MembersWhoHaveRead!.Count 
+                            @if(book.MembersWhoHaveRead.Count == 1)
+                            {
+                                <a>Member</a>
```
Comment (ardalis) [inline src/DevBetterWeb.Web/Pages/Leaderboard/Index.cshtml:66]: Why did you wrap this in <a> tags if it isn't a link? If you just needed Razor to understand it's HTML content not C# code, use a <span> or read this thread:
https://stackoverflow.com/questions/27284734/render-a-string-as-html-in-c-sharp-razor (don't use Html.Raw)

File: src/DevBetterWeb.Web/Pages/Leaderboard/Index.cshtml.cs [hunk 2]
```diff
@@ -49,7 +49,9 @@ public async Task OnGet()
           .ToList();
 
       var books = await _appDbContext.Books.AsQueryable()
-        .OrderBy(book => book.Title)
+        .OrderByDescending(book => book.MembersWhoHaveRead.Count)
+        .ThenBy(book => book.Title)
```
Comment (ardalis) [inline src/DevBetterWeb.Web/Pages/Leaderboard/Index.cshtml.cs:53]: Looks good!

**Review by ardalis — CHANGES_REQUESTED** [review]
Fix <a> then good to go.

**Comment by ilyanaDev** [comment]
Fixed

**Review by ardalis — APPROVED** [review]

**Pull request merged**